README.md

```diff
@@ -4,71 +4,77 @@ The Open Apparel Registry (OAR) is a tool to identify every apparel facility wor
 
 - [Requirements](#requirements)
 - [Setup](#setup)
+  - [Google Maps Platform](#google-maps-platform)
 - [Development](#development)
   - [Hot Reloading 🔥](#hot-reloading-)
-  - [Development Data](#development-data)
   - [Ports](#ports)
-- [Scripts](#scripts)
+- [Scripts 🧰](#scripts-)
 
 ## Requirements
 
-- Vagrant 2.1+
-- VirtualBox 5.0+
-- AWS CLI 1.1+
-- IAM credentials (for artifacts, secrets, etc)
+- [Vagrant](https://www.vagrantup.com/docs/installation/) 2.1+
+- [VirtualBox](https://www.virtualbox.org/wiki/Downloads) 5.0+
 
 ## Setup
 
-First, configure a local AWS profile with access to an S3 bucket with files containing project specific environment variables:
+Run `setup` to bring up the development environment:
 
 ```bash
-$ aws configure --profile open-apparel-registry
-AWS Access Key ID [None]: AKIAIOSFODNN7EXAMPLE
-AWS Secret Access Key [None]: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
-Default region name [None]: eu-west-1
-Default output format [None]:
+./scripts/setup
 ```
 
-Next, use `setup` to bring up the development environment:
+`setup` will provision a virtual machine (VM) that contains the tools needed to get started.
 
-```bash
-$ ./scripts/setup
-```
-
-`setup` will provision a Vagrant VM, using `vboxsf` to mount `./` (relative to `Vagrantfile`) to `/vagrant`, as well as `~/.aws` to `/home/vagrant/.aws`, and execute `update`.
-
-After running `setup`, you can access the VM by running:
+After executing `setup`, you can access the VM with:
 
 ```bash
 $ vagrant ssh
 vagrant@vagrant:/vagrant$
 ```
 
+### Google Maps Platform
+
+The OAR requires a Google Maps Platform API key to interface with the Maps JavaScript API, Maps Static API, and Maps Geocoding API. 
+
+Without an API key, facility detail maps will not load on the client and geocoding will not function on the server. The basemap will also be low-resolution and watermarked with "for development purposes only." Since geocoding is 
```

Comment:
The last sentence in this paragraph appears to be incomplete.

Reply:
Nice catch. I pushed that up after Justin reviewed.